### DISCUSSION TITLE
Benchmark Checkpoints

### DISCUSSION THREAD

**Discussion by arshka** [PolymathicAI/the_well#24] [Q&A] [answered]
Hello,

Thanks for putting this dataset together. I was wondering if the checkpoints for the benchmarks are available somewhere?

Appreciate it!

**Comment by rubenohana** [comment]
Hello, 

Thank you for your message! The checkpoints are currently not public, but if there is interest, we could try to provide some of them. Is there a specific checkpoint that you would need (dataset, model, learning rate)?

Ruben

**Comment by arshka** [comment]
Thanks for responding,

Providing the benchmark model weights would be really helpful for comparing methods against other metrics and timeframes not reported in the paper. The code is provided, but there’s quite a lot of benchmark models to train independently, 4 for each simulation. 

Appreciate it!

**Comment by ChristianInterno** [comment]
the checkpoints of the model for each dataset would be really helpful :)

**Comment by rubenohana** [comment]
Ok, let me see what we can do!

**Comment by LTMeyer** [comment, marked as answer]
The checkpoints of the models benchmarked in the original paper have been uploaded to HF. See [The Well Benchmark Model Collection](https://huggingface.co/collections/polymathic-ai/the-well-benchmark-models-67e69bd7cd8e60229b5cd43e). 

You can now load a specific checkpoint use the following example for the FNO model trained on `active_matter`

```python
from the_well.benchmark.models import FNO

model = FNO.from_pretrained("polymathic-ai/FNO-active_matter")
```
